eng/templates/Azure.ServiceTemplate.Template/.template.config/template.json

```diff
@@ -55,7 +55,8 @@
     "swagger": {
       "type": "parameter",
       "datatype":"text",
-      "isRequired": true,
+      "isRequired": false,
```

Comment:
Is it possible to require that one of swagger or readme is present?

Reply:
NO. There is not rule to define one-of symbols in .template.config